hoot-test/src/main/cpp/hoot/test/main.cpp

```diff
@@ -76,17 +76,37 @@ using namespace std;
 
 typedef boost::shared_ptr<CppUnit::Test> TestPtr;
 
+enum _TestType
+{
+  CURRENT,
+  QUICK,
+  QUICK_ONLY,
+  SLOW,
+  SLOW_ONLY,
+  GLACIAL,
+  GLACIAL_ONLY,
+  SERIAL,
+  ALL
+};
+
+enum _TimeOutValue
+{
+  QUICK_WAIT    = 3,
+  SLOW_WAIT     = 30,
+  GLACIAL_WAIT  = 900
+};
+
```

Comment:
Just moved these up to be able to use them elsewhere but I did change the `QUICK_WAIT` to 3 seconds, up from 2.  There is one quick test that takes about 2 seconds.  Sometimes a touch less and sometimes a skosh more.  This just keeps that test from complaining when it takes 2.001 seconds to complete.